app/views/agency/uscis/index.html.erb

```diff
@@ -42,6 +42,9 @@
       </div>
       <div class="sm-col-right">
         <form action="/auth/saml" method="GET">
+          <% if params[:loa] %>
+            <%= hidden_field_tag 'loa', params[:loa] %>
+          <% end %>
```

Comment:
Now that you've created a shared form partial, shouldn't it be used in each of these agency-specific views?